docs/docs/build/examples/metamask-ordinals/_category_.yml

```diff
@@ -0,0 +1,3 @@
+position: 4
```

Comment:
Please update the other positions accordingly. Preferred order:

![image](https://github.com/bob-collective/bob/assets/14966470/ff281667-09d5-4e2f-a574-02f509d5a8d9)